### PR TITLE
fix: deploy is forced by default

### LINKED ISSUE #211: `sasjs deploy` should always force deploy
when running `sasjs deploy`, the expectation is always that the `appLoc` will be deleted and rebuilt, so the `-f` flag should be applied by default

image

### PR DESCRIPTION
**Pull request by medjedovicm**
## Issue

Closes #211 

## Intent

Deploy is always forced, by default.

## Checks

- [x] Code is formatted correctly (`npm run lint:fix`).
- [x] Any new functionality has been unit tested.
- [x] All unit tests are passing (`npm test`).
- [x] All CI checks are green.
- [x] JSDoc comments have been added or updated.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -111,6 +111,11 @@ export async function compileServices(targetName) {
 }
 
 export async function deployServices(targetName, isForced) {
+  /**
+   * Deploying is forced by default
+   */
+  isForced = true
```
Comment (allanbowe) [inline src/main.js:117]: does this make any sense in the context of the signature in line 113?

Comment (YuryShkoda) [reply src/main.js:117]: If deploy is always forced, then we should delete `isForced` from the `cli` at all.
Instead we should update `deployServicePack` method in `adapter` to:
image

Comment (allanbowe) [reply src/main.js:117]: I agree we should remove isforced from the cli (it's redundant with this change), but not from the adapter - I think it's good to be explicit that we are going to do a full replace of the target appLoc.

Comment (medjedovicm) [reply src/main.js:117]: Makes sense

**Review by allanbowe — APPROVED** [review]

**Review by YuryShkoda — APPROVED** [review]

File: src/commands/deploy.js [hunk 2]
```diff
@@ -92,7 +88,7 @@ export async function deploy(
       // split into lines
       const linesToExecute = deployScriptFile.replace(/\r\n/g, '\n').split('\n')
       if (targetToBuild.serverType === 'SASVIYA') {
-        await deployToSasViya(
+        await deployToSasAViya(
```
Comment (allanbowe) [inline src/commands/deploy.js:91]: is this a typo?

File: src/commands/deploy.js [hunk 3]
```diff
@@ -194,11 +190,11 @@ async function deployToSasViyaWithServicePack(buildTarget, isForced) {
       null,
       null,
       access_token,
-      isForced
+      true
     )
   }
 
-  return await sasjs.deployServicePack(jsonObject, null, null, null, isForced)
+  return await sasjs.deployServicePack(jsonObject, null, null, null)
```
Comment (saadjutt01) [inline src/commands/deploy.js:197]: In this case, we are not forcing deploy? it's intentional?

**Review by saadjutt01 — DISMISSED** [review]

**Review by krishna-acondy — DISMISSED** [review]

**Pull request merged**